src/compiler/python_generator.cc

```diff
@@ -53,8 +58,11 @@ using google::protobuf::Descriptor;
 using google::protobuf::FileDescriptor;
 using google::protobuf::MethodDescriptor;
 using google::protobuf::ServiceDescriptor;
+using google::protobuf::compiler::GeneratorContext;
+using google::protobuf::io::CodedOutputStream;
 using google::protobuf::io::Printer;
 using google::protobuf::io::StringOutputStream;
+using google::protobuf::io::ZeroCopyOutputStream;
```

Comment:
It's fine for now, but we should use the compiler/config.h file to use grpc::protobuf instead. For a future PR that is.